src/helpers/figures/charts/pie_chart.ts

```diff
@@ -212,6 +213,23 @@ function getPieConfiguration(
   config.options.plugins!.tooltip!.callbacks!.title = function (tooltipItems) {
     return tooltipItems[0].dataset.label;
   };
+  config.options.plugins!.tooltip!.callbacks!.label = function (tooltipItems) {
+    const { format, locale } = localeFormat;
+    const data = tooltipItems.chart.data;
+    const percentageValues = calculatePercentageValues(data.datasets);
```

Comment:
Use `tooltipItems.dataSet` maybe ? And what's the point of computing the percentages values for both datasets ? We only need the valeus for the current dataset. And we only need the value of the current `dataIndex`, we don't care about the others, it's useless to compute them.